index.html

```diff
@@ -0,0 +1,424 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/38b6d4ab77.js"
+      crossorigin="anonymous"
+    ></script> 
+    <link rel="stylesheet" href="./css/style.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Meera+Inimai&family=Poppins&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Zunair Desktop Portfolio Website</title>
+  </head>
+
+  <body>
+    <div class="container">
+
+      <header class="header-section">
+
+        <nav class="nav-bar">
+          <div class="left-navbar">
+            <a href="#" class="logo">Muhammad Zunair khan</a>
+            <div class="wrap">
+              <div class="menu"></div>
+              <div class="menu"></div>
+              <div class="menu"></div>
+            </div>
+          </div>
+  
+          <div class="right-navbar">
+            <ul>
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+              <li><a href="#"><i class="fa fa-envelope-o"></i></a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
```

Comment:
- [ ] As the previous reviewer suggested, kindly ensure that there is no horizontal scroll and that the nav bar does not break. This is because the horizontal scroll can stress the users and feel uncomfortable with the page.

Your design 😎 
![213795570-84c9424f-81c6-46ad-ac65-8858a5a49d15](https://user-images.githubusercontent.com/99394374/213868717-9c3ef3b9-94a5-45b3-9ec5-2ce747289d15.png)

Reply:
I have removed the scroll